### PR TITLE
adding start_html text to the sync in

### PR DESCRIPTION
**Pull request by annaxuphoto**
This PR adds the `start_html` attribute in starter and sample apps to the i18n sync. We do so by extracting the text content from the XML nodes and adding the strings to the sync, to be keyed by their English value.

## Links
Jira: https://codedotorg.atlassian.net/browse/FND-2119

## Testing story

Ran the sync locally to verify the correct behavior. Some examples of the strings that were added:

Screen Shot 2022-11-28 at 5 40 11 PM
Screen Shot 2022-11-28 at 5 40 20 PM
Screen Shot 2022-11-28 at 5 40 30 PM

## Follow-up work

Sync out changes and testing to verify the correct translations show on Dashboard.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: bin/i18n/sync-in.rb [hunk 1]
```diff
@@ -243,6 +243,18 @@ def get_i18n_strings(level)
...
     end
 
+    # start_html
+    if level.start_html
+      start_html = Nokogiri::XML(level.start_html, &:noblanks)
+      i18n_strings['start_html'] = Hash.new unless level.start_html.empty?
+
+      # match any element that contains text
+      start_html.xpath('//*[text()[normalize-space()]]').each do |element|
+        puts "element: " + element.text
```
Comment (wilkie) [inline bin/i18n/sync-in.rb]: I assume this would produce a lot of console output!

Comment (annaxuphoto) [reply bin/i18n/sync-in.rb]: haha! i thought i took this out dang

Comment (annaxuphoto) [reply bin/i18n/sync-in.rb]: probably accidentally put it back when i was messing with the git base branch

Comment (annaxuphoto) [reply bin/i18n/sync-in.rb]: fixed!

**Review by wilkie — APPROVED** [review]
Looks good. To sum up, this will place the text nodes from the HTML via `start_html` into the i18n strings under `start_html`. This makes sense... you'll just do a lookup for all text nodes at some point during initial render of the page.

I think I would remove the `puts` and then just consider it approved and merge it.

Nice work! 👍 Thank you for this!

**Pull request merged**